src/test/java/io/jenkins/plugins/grading/CoverageScoreTests.java

```diff
@@ -14,15 +12,45 @@
  *
  * @author Eva-Maria Zeintl
  * @author Ullrich Hafner
+ * @author Andreas Riepl
  */
 class CoverageScoreTests {
+
     @Test
-    void shouldCalculate() {
+    void givenCoverageScore_whenRatio99Percent_thenTotalImpactShouldEqualImpactValue() {
+        int missedImpactValue = -2;
+
         CoverageConfiguration coverageConfiguration = new CoverageConfigurationBuilder()
-                .setMissedImpact(-2)
+                .setMissedImpact(missedImpactValue)
                 .build();
         CoverageScore coverageScore = new CoverageScore(coverageConfiguration, Ratio.create(99, 100));
 
-        assertThat(coverageScore).hasTotalImpact(-2);
+        assertThat(coverageScore).hasTotalImpact(missedImpactValue);
+    }
+
+    @Test
+    void givenCoverageScore_whenRatio90Percent_thenTotalImpactShouldBeTenTimesHigher() {
+        int missedImpactValue = 6;
+
+        CoverageConfiguration coverageConfiguration = new CoverageConfigurationBuilder()
+                .setMissedImpact(missedImpactValue)
+                .build();
+        CoverageScore coverageScore = new CoverageScore(coverageConfiguration, Ratio.create(90, 100));
+
+        assertThat(coverageScore).hasTotalImpact(missedImpactValue*10);
```

Comment:
Leider ist gerade Codacy noch nicht aktiv für unser Projekt: hier mein Styleguide: https://github.com/uhafner/codingstyle/blob/master/doc/Formatierung.md

Haben Sie das Checkstyle Plugin in IntelliJ aktiv, das sollte dies als Warnung anzeigen.

Reply:
Plugin ist jetzt aktiv